src/views/edit/edit-view.jsx

```diff
@@ -305,6 +308,39 @@ export default function EditView({ panel }) {
 		};
 	}, [compareToContact, editId, existingContact, keys?.length, panel]);
 
+	const title = useMemo(
+		() =>
+			contact?.namePrefix ||
+			contact?.firstName ||
+			contact?.middleName ||
+			contact?.nickName ||
+			contact?.lastName ||
+			contact?.nameSuffix
+				? `${contact?.namePrefix ? `${contact?.namePrefix} ` : ''}${
+						contact?.firstName ? `${contact?.firstName} ` : ''
+				  }${contact?.middleName ? `${contact?.middleName} ` : ''}${
+						contact?.nickName ? `${contact?.nickName} ` : ''
+				  }${contact?.lastName ? `${contact?.lastName} ` : ''}${
+						contact?.nameSuffix ? `${contact?.nameSuffix} ` : ''
+				  }`
+				: t('label.new_contact', 'New contact'),
```

Comment:
```suggestion
				? `${contact?.namePrefix ?? ''
				}${contact?.firstName ?? ''
				}${contact?.middleName ?? ''
				}${contact?.nickName ?? ''
				}${contact?.lastName ?? ''
				}${contact?.nameSuffix ?? ''}`
				: t('label.new_contact', 'New contact'),
```

Html eats out the excess spaces around the string and also inside. Me and Giuliano checked it out also with the Text component in the dev docs and it works.